golang/src/com/google/idea/blaze/golang/run/BlazeGoRunConfigurationRunner.java

```diff
@@ -141,7 +141,7 @@ private ImmutableList<String> getParameters(ExecutableInfo executable) {
       return ImmutableList.<String>builder()
           .addAll(state.getExeFlagsState().getFlagsForExternalProcesses())
           .addAll(state.getTestArgs())
-          .addAll(executable.args)
+          .addAll(state.getUserEnvVarsState().asBlazeTestEnvFlags())
```

Comment:
I think `ExecutableInfo` already has a field `envVars`, will it contain different variables from `state.getUserEnvVarsState().asBlazeTestEnvFlags()`?

Why do we no longer need the `executable.args`? and also we can remove the `executable` parameter of this function if it is not needed.

Reply:
The envVars from `ExecutableInfo` are just for the test runner itself, unrelated to the ones the user specifies. However, this comment uncovers a massive bug in the implementation, thanks for pointing it out. I'll fix it.